pages/project-registry/integrating-with-the-project-registry.mdx

```diff
@@ -0,0 +1,187 @@
+import { Tabs, Tab } from 'nextra/components'
+
+# Integrating with the Project Registry
+There are two ways to interact with the Project Registry. 
+1. Through the `Registry.sol` contract
+2. Through the Allo subgraph
+
+## Writing to the Project Registry
```

Comment:
You can remove this section, since we have a page that lists all the contracts across chains. We'll only want to keep that info up to date in one place